code/src/App.js

```diff
@@ -1,9 +1,127 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
```

Comment:
Is there a reason you chose to have all the survey questions located in app.js as opposed to in separate components?